tests/sles4sap/saptune.pm

```diff
@@ -26,6 +26,10 @@ sub run {
 
     select_console 'root-console';
 
+    unless (tuned_is 'running') {
+        script_run "saptune daemon start";
```

Comment:
That should not be allowed to fail, is it? I recommend by default to use `assert_script_run` or explicitly check the return value if you want to act on it differently

Reply:
You're correct